css/style.css

```diff
@@ -0,0 +1,1139 @@
+    color: #f1f1f1;
+    width: 100%;
+    padding: 15px 0px;
+    display: flex;
+    align-items: center;
+    justify-content:space-evenly;
+    transition: all 2s ease; 
+    transform:translatey(60px);
+}
+
+.today__img__fond__static{
+    position: absolute;
+    bottom: 0;
+    background: rgba(0, 0, 0, 0.5);
+    color: #f1f1f1;
+    width: 100%;
+    padding: 15px 0px;
+    display: flex;
+    align-items: center;
+    justify-content:space-evenly;
+    /* transition: all 2s ease; 
+    transform:translatey(60px); */
+}
+
+.today__img:hover .today__img__fond{
+    transition: all 0.6s ease; 
+    transform:translatey(0px);
+}
+
+.today__img__fond__title{
+    justify-content:flex-start;
+    justify-self: start;
+    justify-items: flex-start;
+    padding-right: 100px;
+    padding-left: 15px;
+}
+
+.today__img__fond__number{
+    padding-left: 10px;
+}
+
+.today__img__fond__comment{
+    margin-bottom: -5px;
+    margin-left: 2px;
+}
+
+/* -------------------------- */
+
+.today__descript{
+    color: black;
+    margin-top: 40px;
+    margin-bottom: 40px;
+}
+
+.today__button {
+    padding: 12px 19px;
+    color: white;
+    font-size: 16px;
+    left: 5.56%;
+    right: 86.25%;
+    top: 68.79%;
+    bottom: 30.47%;
+    background: #006DFE;
+}
+
+/* ================================== Today end =============================== */
+/* |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||| */
+
+/* ================================ Video start =============================== */
+
+.videograf {
+    position: relative;
+    margin-top: 100px;
+    margin-bottom: -5px;
+}
+
+.videograf video {
+    /* width:-moz-available; */
+    width: 100%;
+}
+
+.videograf__logo {
+    position: absolute;
+    top:5%;
+    left:3.3%;
+    background: rgba(31, 136, 36, 0.5);
+    border-radius: 48%;
+    width: 15%;
+    padding: 1% 1%;
+}
+
+/* ================================= Video end =============================== */
+/* |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||| */
+
+/* ================================ Footer start =============================== */
+
+footer {
+    background-color: #0E1D28;  
+    display: flex;
+    flex-wrap:wrap;
+    justify-content:space-between;    
+    align-items:flex-start;    
+}
+
+.footer__logo {
+    margin: 120px 150px 0px 220px;
+    width: 175px;
+    height: 65px;
+}
+
+/* ---------------- */
+
+footer ul {
+    margin: 120px 20px 120px 20px;
+    min-width: 115px;
+ }
+ 
+ footer li {
+    list-style-type:none;
+    padding:5px 10px;
+ }
+ 
+/* ---------------- */
+
+footer li a {
+    position: relative;
+    color: #fdfdfd;
+    /*  font-size: 1.25rem; */
+    text-decoration: none;
+      font-size:16px;
+}
+  
+footer li a::before {
+    content: '';
+    position: absolute;
+    left: 0; bottom: -5px;
+    width: 0;
+    height: 1.5px;
+    background-color: #1d9c23;
+}
+  
+footer li a:hover:before {
+    width: 100%;

```

Comment:
Looks like you have the same underlying functional for header and footer section. Instead of copy&paste I think you can create common styles and reuse them